src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -63,6 +63,44 @@ public CommandResult execute(Model model, ActiveAccount activeAccount) throws Co
             activeAccount.updateFilteredRevenueList(revenuePredicate);
         }
 
-        return CommandResultFactory.createDefaultCommandResult(Messages.MESSAGE_ENTRIES_LISTED_OVERVIEW);
+        boolean noExpenseFoundWithExpensePredicate =
+            activeAccount.getFilteredExpenseList().size() == 0 && expensePredicate != null;
+        boolean noRevenueFoundWithRevenuePredicate =
+            activeAccount.getFilteredRevenueList().size() == 0 && revenuePredicate != null;
+        boolean noEntryFound = activeAccount.getFilteredExpenseList().size() == 0
+                            && activeAccount.getFilteredRevenueList().size() == 0;
+
+        if (noEntryFound || noExpenseFoundWithExpensePredicate || noRevenueFoundWithRevenuePredicate) {
```

Comment:
What if you set the predicate of an empty revenue list, then you set a predicate to the expense list. Would this result in a Empty message?